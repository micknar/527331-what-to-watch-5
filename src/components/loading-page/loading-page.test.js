import React from "react";
import renderer from "react-test-renderer";
import LoadingPage from "./loading-page";

it(`Should LoadingPage render correctly`, () => {
  const tree = renderer
    .create(
        <LoadingPage />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
