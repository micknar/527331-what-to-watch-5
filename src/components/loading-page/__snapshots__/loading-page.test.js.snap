// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should LoadingPage render correctly 1`] = `
<div
  className="user-page"
>
  <header
    className="page-header user-page__head"
  >
    <div
      className="logo"
    >
      <div
        className="logo__link"
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </div>
    </div>
  </header>
  <div
    className="sign-in user-page__content"
  >
    <div
      className="sign-in__message"
      style={
        Object {
          "display": "flex",
          "justifyContent": "center",
        }
      }
    >
      <div
        dangerouslySetInnerHTML={
          Object {
            "__html": "<svg xmlns=\\"http://www.w3.org/2000/svg\\" viewBox=\\"0 0 32 32\\">
  <path transform=\\"translate(2)\\" d=\\"M0 12 V20 H4 V12z\\"> 
    <animate attributeName=\\"d\\" values=\\"M0 12 V20 H4 V12z; M0 4 V28 H4 V4z; M0 12 V20 H4 V12z; M0 12 V20 H4 V12z\\" dur=\\"1.2s\\" repeatCount=\\"indefinite\\" begin=\\"0\\" keytimes=\\"0;.2;.5;1\\" keySplines=\\"0.2 0.2 0.4 0.8;0.2 0.6 0.4 0.8;0.2 0.8 0.4 0.8\\" calcMode=\\"spline\\"  />
  </path>
  <path transform=\\"translate(8)\\" d=\\"M0 12 V20 H4 V12z\\">
    <animate attributeName=\\"d\\" values=\\"M0 12 V20 H4 V12z; M0 4 V28 H4 V4z; M0 12 V20 H4 V12z; M0 12 V20 H4 V12z\\" dur=\\"1.2s\\" repeatCount=\\"indefinite\\" begin=\\"0.2\\" keytimes=\\"0;.2;.5;1\\" keySplines=\\"0.2 0.2 0.4 0.8;0.2 0.6 0.4 0.8;0.2 0.8 0.4 0.8\\" calcMode=\\"spline\\"  />
  </path>
  <path transform=\\"translate(14)\\" d=\\"M0 12 V20 H4 V12z\\">
    <animate attributeName=\\"d\\" values=\\"M0 12 V20 H4 V12z; M0 4 V28 H4 V4z; M0 12 V20 H4 V12z; M0 12 V20 H4 V12z\\" dur=\\"1.2s\\" repeatCount=\\"indefinite\\" begin=\\"0.4\\" keytimes=\\"0;.2;.5;1\\" keySplines=\\"0.2 0.2 0.4 0.8;0.2 0.6 0.4 0.8;0.2 0.8 0.4 0.8\\" calcMode=\\"spline\\" />
  </path>
  <path transform=\\"translate(20)\\" d=\\"M0 12 V20 H4 V12z\\">
    <animate attributeName=\\"d\\" values=\\"M0 12 V20 H4 V12z; M0 4 V28 H4 V4z; M0 12 V20 H4 V12z; M0 12 V20 H4 V12z\\" dur=\\"1.2s\\" repeatCount=\\"indefinite\\" begin=\\"0.6\\" keytimes=\\"0;.2;.5;1\\" keySplines=\\"0.2 0.2 0.4 0.8;0.2 0.6 0.4 0.8;0.2 0.8 0.4 0.8\\" calcMode=\\"spline\\" />
  </path>
  <path transform=\\"translate(26)\\" d=\\"M0 12 V20 H4 V12z\\">
    <animate attributeName=\\"d\\" values=\\"M0 12 V20 H4 V12z; M0 4 V28 H4 V4z; M0 12 V20 H4 V12z; M0 12 V20 H4 V12z\\" dur=\\"1.2s\\" repeatCount=\\"indefinite\\" begin=\\"0.8\\" keytimes=\\"0;.2;.5;1\\" keySplines=\\"0.2 0.2 0.4 0.8;0.2 0.6 0.4 0.8;0.2 0.8 0.4 0.8\\" calcMode=\\"spline\\" />
  </path>
</svg>
",
          }
        }
        style={
          Object {
            "fill": "#dfcf77",
            "height": "20%",
            "width": "20%",
          }
        }
      />
    </div>
  </div>
  <footer
    className="page-footer"
  >
    <div
      className="logo"
    >
      <a
        className="logo__link logo__link--light"
      >
        <span
          className="logo__letter logo__letter--1"
        >
          W
        </span>
        <span
          className="logo__letter logo__letter--2"
        >
          T
        </span>
        <span
          className="logo__letter logo__letter--3"
        >
          W
        </span>
      </a>
    </div>
    <div
      className="copyright"
    >
      <p>
        Â© 2019 What to watch Ltd.
      </p>
    </div>
  </footer>
</div>
`;
